src/uct/ib/rc/base/rc_ep.h

```diff
@@ -10,6 +10,7 @@
 #include "rc_iface.h"
 
 #include <uct/api/uct.h>
+#include <ucs/debug/debug.h>
```

Comment:
why debug.h added ? If the new macro depends on debug probably the include should go to uct_iface.h